### PR TITLE
Managing help offers

### LINKED ISSUE #2170: Managing help
The thread in #1781 is getting bigger and bigger. A lot of people want to help but seem not willing to RTFM (read the manual). I propose we copy the important parts of #1781 into a new markdown-page in the misc folder and close the issue. The "We need Help" links should then point to the new page.

The main topics should be:
* What is the current status and the role of almende.
* Wow can a beginner helping out (e.g. by answering questions and closing old issues)
* Wow to contribute with Pull-Requests
* Who ist maintaining the project at the moment?
* What are the goals in for the future?

### PR DESCRIPTION
**Pull request by mojoaxel**
This is trying to fix #2170 and to close #1781 

I created a document describing the current maintainer situation (misc/we_need_help.md). 

I also added a "HowTo Help" document (misc/how_to_help.md) that describes in detail how to start getting involved in the project.

Also I adapted the CONTRIBUTIONS document and made it clear that general questions should be asked on stackoverflow.

This docs should be a first start and I hope you all keep them up to date and helpfull!

### CODE REVIEW

**Comment by mojoaxel** [comment]
@ludost Can you please have a look at this pull-request. This is clearly in the realm of the maintainer and I want to make sure we are on the same page here!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,22 +1,13 @@
...
-- When implementing new features, please update the documentation accordingly.
-- Make changes in the `develop` branch, not the `master` branch.
-- Send pull requests to the `develop` branch, not the `master` branch.
-- Only commit changes done in the source files under `lib`, not to the builds
-  which are located in the folder `dist`.
-
-Thanks!
+### Bugs, Problems and Feature-Requests
+If you really want to open a new issue:
+* Please make sure to **mention witch module** of vis.js (network, timeline, graph3d, ...) your are referring to.
```
Comment (yotamberk) [inline CONTRIBUTING.md]: small misspelling "witch" -> "which"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,22 +1,13 @@
...
-  [here](http://nodeguide.com/style.html).
-- When implementing new features, please update the documentation accordingly.
-- Make changes in the `develop` branch, not the `master` branch.
-- Send pull requests to the `develop` branch, not the `master` branch.
-- Only commit changes done in the source files under `lib`, not to the builds
-  which are located in the folder `dist`.
-
-Thanks!
+### Bugs, Problems and Feature-Requests
+If you really want to open a new issue:
```
Comment (yotamberk) [inline CONTRIBUTING.md]: First step should probably be something like: "Please check if the question was asked in the open/closed issues"

File: misc/how_to_help.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+The visjs.org webpage is hosted on the [gh-pages branch](//github.com/almende/vis/tree/gh-pages). If you find a typo or anything else that should be improved feel free to create a pull-request to *gh-pages*. Please make changes in your own fork of gh-pages so the support team can view the changes in your hosted fork.
+
+### Create new examples
+
+We have [a collection of examples](//github.com/almende/vis/tree/develop/examples). Please help by creating interesting new ones that show a specific problem or layout. Keep the examples easy to understand for beginners and remove unnecessary clutter.
+
+### Provide interesting showcases
+
+If you use vis.js to develop something beautiful feel fee to create a pull-request to our show cases page in the gh-pages branch](//github.com/almende/vis/tree/gh-pages/showcase). [These showcases are displayed on our webpage](http://visjs.org/showcase/index.html) and we are always looking for new examples.
```
Comment (yotamberk) [inline misc/how_to_help.md]: Small typeL fee -> free

**Comment by ludost** [comment]
General approach is excellent. I do wonder if you are already willing to be mentioned in the 'current status' as part of the support team? Something like: *a number of people from the user community have been picking up the first work, forming a support team. Currently this team consists of @mojoaxel and @yotamberk .* What do you think about this? 

Besides the issue tracker and the 'enhancement' label, we might want to create some form of roadmap file as well? This file would be tracking longterm goals in the development. Would something like that be helpful? 

On a fully separate note: it has been a long standing irritation that the /dist folder is actually under version control, leading to local changes that should not be committed until a release is done. I like to know if there are examples out there of how to handle this cleaner? This question influences the development flow, that's why I mentioned it now. 

But again, great work! I'm glad you are helping eve.js forward!

**Comment by mojoaxel** [comment]
> General approach is excellent. 

Thanks! I just thought we need to get rid of #1781

> I do wonder if you are already willing to be mentioned in the 'current status' as part of the support team? 

Why not!

> Besides the issue tracker and the 'enhancement' label, we might want to create some form of roadmap file as well? This file would be tracking longterm goals in the development. Would something like that be helpful?

I don't think that is easy. In a community driven project contributors are normally not willing to stick to a roadmap, but contribute what they want, when the want. As far as I'm concerned a roadmap makes only sense almende internal, if you find time to get somebody implementing again.

> On a fully separate note: it has been a long standing irritation that the /dist folder is actually under version control, leading to local changes that should not be committed until a release is done. I like to know if there are examples out there of how to handle this cleaner? This question influences the development flow, that's why I mentioned it now.

I'm so with you!! I was already looking into it and could not find an easy solution, yet. I created #2181 for that.

**Pull request merged**